app/modules/visualization/js/objectBarsMultiple.js

```diff
@@ -256,11 +481,31 @@ function wrap(text, width) {
         	//var range = 10;
         	return range;
         })
+		.attr("class", function (d, i) {
+			
+			var find = d.Key;
+			/*
+			var res = find.split("_");
+			var re = '';
+			if (res.length>0) {
+				re = res[res.length-1];
+			}				
+			*/
+			var re = find.replace(/ /g, '_');
+			re = re.replace(/\'/g, '_');
+			re = re.replace(/"/g, '_');
+			re = re.replace(/,/g, '_');
+			re = re.replace(/\[/g, '_');
+			re = re.replace(/\]/g, '_');

```

Comment:
these replacement blocks that you repeat can be replaced with a single line:
var re = find.replace(/[ '",[\]]/g,"_")
But since this is a class name, you can use:
var re = find.replace(/[^\w\-\u00A0-\uFFFFF]/g,"_")
which will replace all characters except the ones valid for css class names: characters [a-zA-Z0-9] and ISO-10646 (Unicode) characters U+00A0 and higher, plus the hyphen (-) and the underscore (_)